crates/net/network/src/session/active.rs

```diff
@@ -181,6 +181,19 @@ impl ActiveSession {
             EthMessage::NewPooledTransactionHashes(msg) => {
                 self.try_emit_broadcast(PeerMessage::PooledTransactions(msg)).into()
             }
+            EthMessage::NewPooledTransactionHashes68(msg) => {
+                if msg.hashes.len() != msg.types.len() || msg.hashes.len() != msg.sizes.len() {
```

Comment:
@Rjected perhaps we should apply this check in the stream directly? so the stream only yields well-formatted messages

Reply:
yes, we should be doing these checks in the stream, and malformed messages should be considered invalid